torch_xla/csrc/xla_graph_executor.cpp

```diff
@@ -76,6 +76,34 @@ bool ShouldSyncIrValue(const torch::lazy::Value& ir_value) {
   return ir_value->op() != xla_not_supported;
 }
 
+XLAGraphExecutor::ComputationCache* CreateComputationCache() {
+  static const size_t kMaxCacheSize =
+      runtime::sys_util::GetEnvInt("XLA_COMPILATION_CACHE_SIZE", 1024);
+  static const bool readonlyPersistentCache =
+      runtime::sys_util::GetEnvBool("XLA_PERSISTENT_CACHE_READ_ONLY", false);
+  static std::string persistentCacheDir =
+      runtime::sys_util::GetEnvString("XLA_PERSISTENT_CACHE_PATH", "");
+  if (!persistentCacheDir.empty()) {
+    auto serialize_fn = [](auto& computation) -> std::string {
```

Comment:
nit: can you fill in these `auto`s? In particular, I can't tell what type `computation` is supposed to be

Reply:
I learned while filling in the `auto`s that I also didn't know what type `computation` was lol